virtualizarr/translators/kerchunk.py

```diff
@@ -1,17 +1,96 @@
 from typing import Any, Mapping, MutableMapping, cast
 
+import numpy as np
 from xarray import Dataset
 from xarray.core.indexes import Index
 from xarray.core.variable import Variable
+from zarr.core.common import JSON
+from zarr.core.metadata import ArrayV3Metadata
+from zarr.core.metadata.v2 import ArrayV2Metadata
 
+from virtualizarr.codecs import (
+    numcodec_config_to_configurable,
+)
 from virtualizarr.manifests import ChunkManifest, ManifestArray
 from virtualizarr.manifests.manifest import ChunkEntry, ChunkKey
+from virtualizarr.manifests.utils import create_v3_array_metadata
 from virtualizarr.readers.common import separate_coords
 from virtualizarr.types.kerchunk import (
     KerchunkArrRefs,
     KerchunkStoreRefs,
 )
-from virtualizarr.zarr import ZArray, ZAttrs, determine_chunk_grid_shape
+from virtualizarr.utils import determine_chunk_grid_shape
+
+
+def to_kerchunk_json(v2_metadata: ArrayV2Metadata) -> str:
```

Comment:
to compare with https://github.com/zarr-developers/VirtualiZarr/blob/main/virtualizarr/zarr.py#L129-L135

Reply:
This function is a bit more complex than it was before because we are going from zarr.core.metadata.v2.ArrayV2Metadata to json (rather than from ZArray.dict() to json). So we have to convert the filters and compressor to serializable objects using `get_config`.